From dff37259e39677a3478cb3172ddd62e747ab86d2 Mon Sep 17 00:00:00 2001
From: Valera <valera@example.com>
Date: Mon, 21 Sep 2026 14:39:18 +0300
Subject: [PATCH 9/9] sm8350-common: Disable SSR ramdumps and modem diagnostic
 logging

- persist.vendor.ssr.enable_ramdumps=0
- persist.sys.ssr.enable_ramdumps=0
- persist.vendor.sys.modem.logging.enable=false
- persist.vendor.radio.snapshot_enabled=0

Change-Id: Ie032320819730934bf6cd52861fc65587405e3ac
---
 vendor.prop | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/vendor.prop b/vendor.prop
index 95199ff..248d46f 100644
--- a/vendor.prop
+++ b/vendor.prop
@@ -291,3 +291,10 @@ ro.config.media_vol_steps=18
 persist.vendor.verbose_logging_enabled=false
 persist.vendor.service.bdroid.snooplog=false
 persist.vendor.service.bdroid.fwsnoop=false
+
+# Disable Subsystem Ramdumps and Modem diagnostic logging
+persist.vendor.ssr.enable_ramdumps=0
+persist.sys.ssr.enable_ramdumps=0
+persist.vendor.sys.modem.logging.enable=false
+persist.vendor.radio.snapshot_enabled=0
+
-- 
2.55.0

